src/send_data_bak.py

```diff
@@ -0,0 +1,71 @@
+from sensor_temp import get_temp
```

Comment:
Don't add backup files in PR.
If you want keep it in your local, Add `*_bak.py` in `.gitignore`

Reply:
Ok, but then we won't have the backup saved anywhere if something happens to our computer.

Reply:
Added to gitignore but probably too late.  It still shows up.

Reply:
We use `git`. `git` will track all the files that you committed.